### DISCUSSION TITLE
Adding custom fonts to Index.scss

### DISCUSSION THREAD

**Discussion by chilleregeravi** [vuepress-theme-hope/vuepress-theme-hope#2698] [Q&A] [answered]
Hi All, 

I am trying to add my custom fonts as mentioned in the documentation to index.scss file under styles. The code I used

@font-face {
    font-family: 'ABCD';
    font-style: normal;
    src: url('pathtothefont') format('woff');

during the build process a message is generated that the font didn't resolve at build time, it will remain unchanged to be resolved at runtime. My understanding is that, the styles are applied at run-time. However, my fonts are not copied over into the final dist build and are not available at runtime. What am I missing here?

**Comment by chilleregeravi** [comment, marked as answer]
Fixed it, I just copied the files into public/assets folder and it worked.

**Comment by Mister-Hope** [comment]
I just saw this, I am not checking my mail because of lunar new year and I am on vacation. Alias like `url(@source/.vuepress/styles/xxx)` and relative path like `url(./xxx)` should be supported.
